spec.html

```diff
@@ -43,15 +43,15 @@
   }
 </style>
 <script>
```

Comment:
This entire script block is obsolete I think. I suggest removing it as per #1575 and then I can close that.

Reply:
Done. (Also, sorry, I should have seen your PR earlier.)